modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -176,6 +182,10 @@ const toggleFullscreen = (editor: Editor, fullscreenState: Cell<ScrollInfo | nul
     editorContainerStyle.top = fullscreenInfo.containerTop;
     editorContainerStyle.left = fullscreenInfo.containerLeft;
 
+    sinkContainerS.each((elm) => {
+      Css.set(elm, 'position', 'relative');
```

Comment:
could it make sense to store the `position` before line 160, and then put that position back instead of using always `relative`?

Reply:
Yes. The previous sink css position value is preserved within the FullScreenInfo state on [line 150](https://github.com/tinymce/tinymce/blob/c0ed9f3c4a50ee519df6c68365b9f3191b53bf91/modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts#L150)